docs/tutorials/kuai-cli-deploy-contract.md

```diff
@@ -0,0 +1,158 @@
+# How to deploy contracts with kuai-cli
+
+## deploy contract workflow by builtin signer (directly deploy):
+
+1. Create a new conrtact, skip this step if the contract already exists ([How to import exists contract](#how-to-import-contract-to-kuai-project))
+
+- `kuai contract new --name [contract-name]`
+
+```bash
+$ kuai contract new --name always-success
+
+New contract "always-success"
+    Created binary (application) `always-success` package
+Rewrite Cargo.toml
+Rewrite capsule.toml
+Done
+```
+
+2. Build contract
+
+- `kuai contract build --name [contract-name] --release`
+
+```bash
+$ kuai contract build --name always-success --release
+
+Building contract always-success
+   Compiling always-success v0.1.0 (/code/contracts/always-success)
+Done
+```
+
+3. Deploy contract
+
+- `kuai contract deploy --name [contract-name] --from [0x.....] --signer ckb-cli`
+
+```bash
+$ kuai contract deploy --name always-success --from ckt1qyqw8yx5hx6vwcm7eqren0d0v39wvfwdhy3q2807pp --signer ckb-cli
+
+[warn] ANYONE_CAN_PAY script info not found in config!
+[warn] OMNILOCK script info not found in config!
+[warn] `config` changed, regenerate lockScriptInfos!
+The address format generated by generateAddress or scriptToAddress will be deprecated, please migrate to encodeToAddress to generate the new ckb2021 full format address as soon as possible
+Input ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsq0rjr2tndx8vdlvspuehkhkgjhxyhxmjgsqw8hsk's password for sign messge by ckb-cli:
+deploy success, txHash:  0x7507b59b5f715c0fcd83d55df5569d72ff1cb05cd399afe1f366e03918827a58
+```
```

Comment:
```suggestion
  ```bash
  $ kuai contract deploy --name always-success --from 
  ckt1qyqw8yx5hx6vwcm7eqren0d0v39wvfwdhy3q2807pp --signer ckb-cli

  # [warn] ANYONE_CAN_PAY script info not found in config!
  # [warn] OMNILOCK script info not found in config!
  # [warn] `config` changed, regenerate lockScriptInfos!
  # The address format generated by generateAddress or scriptToAddress will be 
  # deprecated, please migrate to encodeToAddress to generate the new ckb2021 full 
  # format address as soon as possible
  # Input ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsq0rjr2tndx8vdlvspuehkhkgjhxyhxmjgsqw8hsk's password for sign messge by ckb-cli:
  # deploy success, txHash:  
  # 0x7507b59b5f715c0fcd83d55df5569d72ff1cb05cd399afe1f366e03918827a58
  ```
```